lib/components/narrative/itinerary-carousel.js

```diff
@@ -99,10 +109,23 @@ class ItineraryCarousel extends Component {
 
 const mapStateToProps = (state, ownProps) => {
   const activeSearch = getActiveSearch(state.otp)
+  let itineraries = null
+  let profileOptions = null
+  let showProfileSummary = false
+  if (activeSearch && activeSearch.planResponse && activeSearch.planResponse.plan) {
+    itineraries = activeSearch.planResponse.plan.itineraries
+  } else if (activeSearch && activeSearch.profileResponse && activeSearch.profileResponse.otp) {
+    profileOptions = activeSearch.profileResponse.otp.profile
+    itineraries = profileOptionsToItineraries(profileOptions)
+    showProfileSummary = true
+  }
```

Comment:
It is really needed to separate the itinerary and profile responses into different objects?